src/data/electionSample.json

```diff
@@ -681,7 +681,7 @@
     },
     {
       "id": "measure-666",
-      "districtId": "district-1",
+      "districtId": "district-666",
```

Comment:
I don't think this change is necessary, and if we make it, we shouldn't point to a non-existent district in the file.

The reason this question doesn't show up, IIRC, is because this question has a `partyId`, so it will only show if a ballot style assigned to that district has a matching `partyId`.

Reply:
fixed!